webpack.config.js

```diff
@@ -14,6 +14,10 @@ module.exports = {
   output: {
     filename: 'py/visdom/static/js/main.js',
   },
+  node: {
+    net: 'empty',
+    dns: 'empty',
```

Comment:
Is this needed because of the `fetch` library?

Reply:
Yup